src/index.ts

```diff
@@ -10,6 +10,7 @@ import {
 import ButtonExample from './button_example';
 import DataBlock from './data_block';
 import Footer from './footer';
+import StyledGrid from './grid';
```

Comment:
why not just `import Grid from './grid';`

Reply:
done